fizzbuzz.py

```diff
@@ -1,5 +1,19 @@
 # -*- coding: utf-8 -*-
 
 
-def fizzbuzz():
-    pass
+def fizzbuzz(limit=100):
+    res = []
+    for i in range(1, limit + 1):
```

Comment:
good point, this was a quick implementation for the workshop, but now with the tests in place we can 'safely' refactor to make it more maintainable